<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NCRDHighChartsItemDialog</class>
 <widget class="QDialog" name="NCRDHighChartsItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HighCharts item settings</string>
  </property>
  <layout class="QVBoxLayout" name="layDia">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Resource files</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblTemplate">
        <property name="text">
         <string>Template File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEditTemplateFile"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnSelectTemplate">
        <property name="text">
         <string>Open...</string>
        </property>
        <property name="icon">
         <iconset resource="designer.qrc">
          <normaloff>:/designer/images/fileopen.png</normaloff>:/designer/images/fileopen.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblResultFile">
        <property name="text">
         <string>Result File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEditResultFile"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnSelectResult">
        <property name="text">
         <string>Save...</string>
        </property>
        <property name="icon">
         <iconset resource="designer.qrc">
          <normaloff>:/designer/images/filesave.png</normaloff>:/designer/images/filesave.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Chart Title</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEditChartTitle"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic chart oprions</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="w1" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="w2" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="w3" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabScript">
      <attribute name="title">
       <string>Option Script</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="NCRDSqlTextEdit" name="scriptEditor"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPreview">
      <attribute name="title">
       <string>Chart Preview</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>785</width>
            <height>390</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="chartPreview">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="NCRDPrintWhenPanel" name="printWhenPanel" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnUpdate">
       <property name="text">
        <string>Update Chart Preview</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NCRDPrintWhenPanel</class>
   <extends>QWidget</extends>
   <header>ncrdprintwhenpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NCRDSqlTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>ncrdsqltextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>scriptEditor</tabstop>
  <tabstop>btnUpdate</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources>
  <include location="designer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NCRDHighChartsItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NCRDHighChartsItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotUpdateChart()</slot>
 </slots>
</ui>
